thirdeye/thirdeye-frontend/app/pods/components/self-serve-alert-yaml-details/template.hbs

```diff
@@ -40,58 +40,49 @@
-      <dt class="col-md-4">
-        Dataset
-      </dt>
-      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.dataset}}>
-        {{if alertData.dataset alertData.dataset "N/A"}}
-      </dd>
-      <dt class="col-md-4">
-        Filtered By
-      </dt>
-      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.filters}}>
-        {{if alertData.filters alertData.filters "N/A"}}
-      </dd>
-      <dt class="col-md-4">
-        Breakdown By
-      </dt>
-      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.dimensionExploration}}>
-        {{if alertData.dimensionExploration alertData.dimensionExploration "N/A"}}
-      </dd>
-    </dl>
-    <dl class="col-xs-12 col-sm-5 row">
-      <dt class="col-md-4">
-        Created By
-      </dt>
-      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.createdBy}}>
-        {{if alertData.createdBy alertData.createdBy "N/A"}}
-      </dd>
-      <dt class="col-md-4">
-        Updated By
-      </dt>
-      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.updatedBy}}>
-        {{if alertData.updatedBy alertData.updatedBy "N/A"}}
-      </dd>
-      <dt class="col-md-4">
-        Last detection
-      </dt>
-      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.lastDetectionTime}}>
-        {{if alertData.lastDetectionTime alertData.lastDetectionTime "N/A"}}
-      </dd>
-      <dt class="col-md-4">
-        Subscribed Groups
-      </dt>
-      <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{subscribedGroups}}>
-        {{if subscribedGroups subscribedGroups "N/A"}}
-      </dd>
-    </dl>
-  </div>
-{{/if}}
\ No newline at end of file
+<div class="te-alert-detail-breakdown row">
+  <dl class="col-xs-12 col-sm-5 row">
+    <dt class="col-md-4">Metric</dt>
+    <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.metric}}>
+      <span>{{if firstMetric firstMetric 'N/A'}}</span>
+      {{#if (gt remainingMetricCount 0)}}
+      <span class="te-search-results__value te-search-results__value--remaining-metric-count">
+        +{{remainingMetricCount}}
+      </span>
+      {{/if}}
+    </dd>
+
+    <dt class="col-md-4">Dataset</dt>
+    <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.dataset}}>
+      {{if alertData.dataset alertData.dataset 'N/A'}}
+    </dd>
+
+    <dt class="col-md-4">Filtered By</dt>
+    <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.filters}}>
+      {{if alertData.filters alertData.filters 'N/A'}}</dd>
+
+    <dt class="col-md-4">Breakdown By</dt>
+    <dd class="col-md-8 te-search-results__value{{valueClassSuffix}}" title={{alertData.dimensionExploration}}>
+      {{if alertData.dimensionExploration alertData.dimensionExploration 'N/A'}}</dd>
```

Comment:
you could format it to set `dd` on the next line.

Reply:
I would refrain from making any manual changes to formatting. It is confusing when exactly to make changes and when not to since I am and we all should be relying on hbs prettier. We want to modify prettify rules instead of making any manual changes as it is counterproductive.

Reply:
Actually you can discard my comment. Rebasing messed up some of the formatting. I have fixed it now.